packages/model/src/working-group.ts

```diff
@@ -8,12 +9,31 @@ export const deliverableStatus = [
 ] as const;
 export type DeliverableStatus = typeof deliverableStatus[number];
 
+export const workingGroupRole = ['Project Manager'] as const;
+export type WorkingGroupRole = typeof workingGroupRole[number];
+
+type WorkingGroupMember = Pick<
+  UserResponse,
+  | 'id'
+  | 'firstName'
+  | 'lastName'
+  | 'displayName'
+  | 'email'
+  | 'avatarUrl'
+  | 'alumniSinceDate'
+> & {
+  workingGroupRole: WorkingGroupRole;
```

Comment:
Ive taken this approach rather than overloading role like team member does. cc: @lctrt